app/src/pages/OnDeviceDisplay/ProtocolDetails/Hardware.tsx

```diff
@@ -76,32 +72,32 @@ const getHardwareName = (protocolHardware: ProtocolHardware): string => {
 
 export const Hardware = (props: { protocolId: string }): JSX.Element => {
   const requiredProtocolHardware = useRequiredProtocolHardware(props.protocolId)
-  const { t } = useTranslation('protocol_details')
+  const { t, i18n } = useTranslation('protocol_details')
 
-  return (
+  return requiredProtocolHardware.length === 0 ? (
+    <EmptySection section="hardware" />
+  ) : (
     <Table>
       <thead>
         <tr>
           <TableHeader>
             <StyledText
-              color={COLORS.darkBlack70}
               fontSize={TYPOGRAPHY.fontSize20}
+              color={COLORS.darkBlack70}
               fontWeight={TYPOGRAPHY.fontWeightSemiBold}
-              lineHeight={TYPOGRAPHY.lineHeight24}
               paddingLeft={SPACING.spacing24}
             >
-              {t('location')}
+              {i18n.format(t('location'), 'sentenceCase')}
```

Comment:
The result is the same but it would be better since this is one word.
```suggestion
              {i18n.format(t('location'), 'capitalize')}
```

Reply:
since the string is 1 word, `capitalize` vs `sentenceCase` won't make a difference but for consistency, it does make sense to use `capitalize`. I will change it for consistency